main/boost/APKBUILD

```diff
@@ -38,89 +58,94 @@ _libs="date_time
 	wserialization
 	"
 for _lib in $_libs; do
+	echo "$_options_carch" | grep -F -q -e "--without-${_lib}" || \
 	subpackages="$subpackages $pkgname-$_lib:_boostlib"
 done
+subpackages="$subpackages $pkgname-bundle:_boost_bundle:noarch"
+
+_set_options() {
+	[ -n "$PY2_VERSION" ] || PY2_VERSION="$(_pyversion python2)"
+	[ -n "$PY3_VERSION" ] || PY3_VERSION="$(_pyversion python3)"
+
+	local _options_python="${PY2_VERSION:+$PY2_VERSION,}${PY3_VERSION}"
+	[ -z "$_options_python" ] || _options_python="python=$_options_python"
+
+	_options="
+		--user-config=\"$builddir/user-config.jam\"
+		--prefix=\"$pkgdir/usr\"
+		release
+		$_options_python
+		toolset=gcc
+		debug-symbols=off
+		threading=single,multi
+		runtime-link=shared
+		link=shared,static
+		cflags=-fno-strict-aliasing
+		--layout=tagged
+		-q
+		-j${JOBS:-2}
+		$_options_carch
+		"
+}
 
 prepare() {
-	default_prepare || return 1
+	default_prepare
 
 	cd "$builddir"
-
-	PY2_VERSION="$(_pyversion python2)"
-	PY3_VERSION="$(_pyversion python3)"
-	abiflags=$(python3-config --abiflags)
-
+	_set_options
+	local abiflags="$(python3-config --abiflags || :)"
```

Comment:
why do you add the trailing `... || :`? I think we want catch error if the command fails instead of just silently ignore it?